cloud/logger/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@raincatcher/logger",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

Comment:
:+1: We'll need to look again into the single versioning in the JIRA for setting up npm publish.